.azure_pipelines/ci-pipeline-code-coverage-nightly.yml

```diff
@@ -76,6 +76,8 @@ jobs:
     
     - template: step-azure-sdk-tests.yml
 
+    - template: step-dotnet-test-suite.yml
```

Comment:
Solutions/Makefile does not list `coreclr`. Is change required in the Makefile to enable coreclr test in code-coverage/nightly/release pipelines?